Dockerfile

```diff
@@ -15,11 +15,15 @@ RUN dnf -y update && \
         wget \
         git \
         python36 \
-        java-1.8.0-openjdk-devel \
+        java-11-openjdk-devel \
         maven \
         cargo \
     && dnf clean all
 
+# ensure we're actually using java 11
+ENV JAVA_HOME=/usr/lib/jvm/java-11-openjdk
+RUN alternatives --set java /usr/lib/jvm/java-11-openjdk/bin/java
```

Comment:
`/usr/lib/jvm/java-11-openjdk` is already a soft link to alternatives, is this command necessary?